.github/workflows/workflow.yml

```diff
@@ -30,33 +30,29 @@ jobs:
         # We don't run tests on all versions of the Windows environment and on
         # 4.02.x and 4.07.x in other environments
         ocaml-compiler:
-          - 4.14.x
+          - 5.1.x
         include:
           # OCaml trunk:
           - ocaml-compiler: ocaml-variants.5.3.0+trunk
             os: ubuntu-latest
             skip_test: true
-          # OCaml 5:
+          # OCaml 4:
```

Comment:
Isn't this a typo? Or should we have a separate 4.14 CI run here?

Reply:
Yes indeed the plan was to add a 4.14 build-only CI instead of the 4.13 one.